### ISSUE TITLE
Print message on 0th rank (`print0`)

### ISSUE THREAD

**Issue by coquelin77** [helmholtz-analytics/heat#739]
**Feature functionality**
Print a message on the 0th rank. This can be used to reduce the amount of print statements in debuging or for verbose options for more complex function

example:
```python
def print0(*args, **kwargs):
    if ht.rank == 0:
        print(*args, **kwargs)
```

**Comment by mtar** [comment]
@coquelin77 Is this issue resolved?

**Comment by ClaudiaComito** [comment]
Yes I think this has been resolved by #816. Closing.
